qa-tests-backend/src/test/groovy/CSVTest.groovy

```diff
@@ -184,7 +184,11 @@ class CSVTest extends BaseSpecification {
     }
 
     def getComponentQuery() {
-        return "COMPONENT ID:" + getComponentId() + "+Fixable:true"
+        return "COMPONENT ID:" + getComponentId() + "+" + isPostgresRun() ? "Is" : "" + "Fixable:true"
+    }
+
+    def getFixableQuery() {
+        return isPostgresRun() ? "Is" : "" + "Fixable:true"
```

Comment:
Why can't we use the same query for PG and nonPG?

```
"" + "Fixable:true"
```
Looks weird

Reply:
All postgres vulnerability graphql resolvers have this line : 
logErrorOnQueryContainingField(query, search.Fixable, "IsFixable")

I am under the impression that these cause this suspicious log in central:
```
graphql/resolvers: 2022/09/01 17:53:17.566701 utils.go:366: Error: Unexpected field (Fixable) found in query to resolver (IsFixable). Response maybe unexpected.
```